src/web/app/pages-student/student-course-details-page/student-course-details-page.component.html

```diff
@@ -111,60 +111,57 @@ <h4 class="font-weight-bold">Self</h4>
             <span>{{student.email}}</span>
           </div>
         </div>
-        <div class="form-group row">
-          <div class="col-4 text-right">
-            <label>Google ID</label>
-          </div>
-          <div class="col-8">
-            <span>{{student.googleId}}</span>
-          </div>
-        </div>
       </div>
     </div>
   </div>
 </div>
 
-<div class="section" *ngIf="teammateProfiles">
+<div class="section" *ngIf="teammateProfiles && teammateProfiles.length">
   <div class="col-sm-12">
-    <h4 class="font-weight-bold">Teammates</h4>
+    <h4 class="font-weight-bold">Team Profiles</h4>
   </div>
   <div class="row" *ngFor="let teammateProfile of teammateProfiles">
     <div class="col-12">
       <div class="row">
         <div class="col-md-2 col-3">
-          <img src="{{getPictureUrl(teammateProfile.pictureKey)}}" class="profile-pic float-left">
+          <img src="{{teammateProfile.photoUrl}}"
+               class="profile-pic float-left" (error)="setDefaultPic(teammateProfile)">
         </div>
         <div class="col-md-10 col-sm-9 col-8">
           <table class="table table-striped">
             <tbody>
             <tr>
               <td class="font-weight-bold">Name</td>
               <td>
-                <span>{{teammateProfile.shortName}}</span>
+                <span *ngIf="teammateProfile.studentProfile.shortName">{{teammateProfile.studentProfile
+                  .shortName}}</span>
               </td>
             </tr>
             <tr>
               <td class="font-weight-bold">Email</td>
               <td>
-                <span>{{teammateProfile.email}}</span>
+                <span *ngIf="teammateProfile.studentProfile.email">{{teammateProfile.studentProfile.email}}</span>
               </td>
             </tr>
             <tr>
               <td class="font-weight-bold">Gender</td>
               <td>
-                <span *ngIf="teammateProfile.gender != Gender.OTHER">{{teammateProfile.gender}}</span>
+                <span *ngIf="teammateProfile.studentProfile.gender != Gender.OTHER">{{teammateProfile.studentProfile
+                  .gender}}</span>
               </td>
             </tr>
             <tr>
               <td class="font-weight-bold">Institution</td>
               <td>
-                <span *ngIf="teammateProfile.institute">{{teammateProfile.institute}}</span>
+                <span *ngIf="teammateProfile.studentProfile.institute">{{teammateProfile.studentProfile
+                  .institute}}</span>
               </td>
             </tr>
             <tr>
               <td class="font-weight-bold">Nationality</td>
               <td>
-                <span *ngIf="teammateProfile.nationality">{{teammateProfile.nationality}}</span>
+                <span *ngIf="teammateProfile.studentProfile.nationality">{{teammateProfile.studentProfile
```

Comment:
I think you can safely remove the `*ngIf`. Check other places also.

Reply:
Yup